packages/x-components/src/x-modules/search/components/__tests__/banner.spec.ts

```diff
@@ -44,28 +44,26 @@ describe('testing Banner component', () => {
     expect(getXComponentXModuleName(wrapper.vm)).toEqual('search');
   });
 
-  it('renders the banner component', () => {
+  it('renders a banner component with title', () => {
     const { wrapper } = renderBanner({
-      banner: {
-        modelName: 'Banner',
-        id: '12345',
-        url: 'https://empathy.co',
-        title: 'Search UIs',
-        image: 'https://empathy.co/x-components.jpg',
-        position: 1,
-        tagging: {
-          click: { url: 'https://track-things.com', params: {} }
-        }
-      }
+      banner: createBannerStub('banner', 'Search UIs')
     });
 
     expect(wrapper.get(getDataTestSelector('banner')).text()).toEqual('Search UIs');
   });
 
+  it('renders a banner component without title', () => {
+    const { wrapper } = renderBanner({
+      banner: createBannerStub('banner')
+    });
+
+    expect(wrapper.get(getDataTestSelector('banner')).text()).toEqual('');
+  });
+
   // eslint-disable-next-line max-len
-  it('emits UserClickedABanner when the user clicks in the left, middle or right button on the component', () => {
+  it('renders a banner which emits UserClickedABanner when the user clicks in the left, middle or right button on the component', () => {
```

Comment:
This test fails, check it again after changing the implementation of the stubs.

Reply:
done